src/cluster/sharding.rst

```diff
@@ -12,290 +12,490 @@
 
 .. _cluster/sharding:
 
-========
-Sharding
-========
+================
+Shard Management
+================
 
 .. _cluster/sharding/scaling-out:
```

Comment:
Is it worth adding a bit of an intro about what this document contains?

Reply:
I added something to this effect:

> This document discusses how sharding works in CouchDB along with how to
> safely add, move, remove, and create placement rules for shards and
> shard replicas.